corehq/apps/domain/forms.py

```diff
@@ -466,6 +467,8 @@ def clean_default_timezone(self):
 
     def save(self, request, domain):
         try:
+            domain.hr_name = self.cleaned_data['hr_name']
+
```

Comment:
This isn't on you to clean up, but as an aside note, this try/except block is horrendous.  It will hide any errors involved in changing a bunch of domain properties, saving users, and writing stuff to files and blames it on the user.  This will probably also leave the domain in a partially changed state if an exception does occur.